examples/24_solar_battery_grid/README.md

```diff
@@ -0,0 +1,87 @@
+# Example 23: Solar Battery Grid System
+
+## Overview
+
+This example demonstrates a solar + battery + grid system that showcases the new unified grid component. The system can both buy electricity from the grid and sell excess electricity back to the grid using separate grid connection instances.
+
+## System Description
+
+### Technologies
+
+1. **Solar PV** (200 MWdc)
+   - Utility-scale solar with single-axis tracking
+   - 1.34 DC/AC ratio
+   - 65% bifacial panels
+   - Ground coverage ratio: 0.3
+
+2. **Battery Storage** (200 MWh, 25 MW charge/discharge rate)
+   - Simple generic storage model
+   - Demand-following control strategy (100 MW demand profile)
+   - Min SOC: 10%, Max SOC: 100%
+   - Charge/discharge efficiency: 100%
+
+3. **Grid Buy** (1000 MW interconnection)
```

Comment:
This is a very large interconnection for the size of the plant. Any reason it's so oversized?